### PR TITLE
task done

### PR DESCRIPTION
**Pull request by ojolanre**
Task 1 contribution done

### CODE REVIEW

**Review by lindaokorie27 — CHANGES_REQUESTED** [review]
The console.log statement is enough. Please remove the return statement

**Comment by ojolanre** [comment]
Done

File: scripts/02248-Olanre.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let fullName = "Ojo Cornelius Lanre";
+let ID = "HNG-02248";
+let language = "Javascript";
+
+function hngTask() {
+	let task = `Hello World, this is ${fullName} with HNGi7 ID ${ID} using ${language} for stage 2 task`;
```
Comment (ifatoki) [inline scripts/02248-Olanre.js]: Please include your email after the word `task` and a space. Look [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700) for details

**Review by ifatoki — CHANGES_REQUESTED** [review]

**Comment by ojolanre** [comment]
Please do check the one i just added, thanks

**Review by lindaokorie27 — APPROVED** [review]

**Review by ifatoki — APPROVED** [review]

**Pull request merged**